src/pal/src/cruntime/misc.cpp

```diff
@@ -293,23 +293,22 @@ void PAL__mm_setcsr(unsigned int i)
 
 #endif // _AMD64_ 
 
-#if defined(_DEBUG)
```

Comment:
Why do we need to enable it for non-debug builds too? If I remember it correctly, the intention was to let the intrinsic memcpy to be used in those builds.